src/main/java/lesson2/concerttickets/service/TicketService.java

```diff
@@ -16,28 +16,28 @@ public TicketService() {
         this.tickets = new HashMap<>();
     }
 
-    public String save(Ticket ticket) {
-        this.tickets.put(ticket.getId(),ticket);
-        return ticket.getId();
+    public String save(Ticket t) {
+        this.tickets.put(t.getId(),t);
+        return t.getId();
     }
 
     public Collection<Ticket> findAll() {
         return tickets.values();
     }
 
-    public static void main(String[] args) {
-        Ticket emptyTicket = new Ticket();
-        Ticket fullTicket = new Ticket("3454A","1234567890",303,
-                System.currentTimeMillis()/1000L,true, StadiumSector.B,
-                20.2,new BigDecimal(150));
-        Ticket limitedTicket = new Ticket("1234567890",305,System.currentTimeMillis()/1000L);
+    public Ticket findById(String ticketId) {
+        if(tickets.containsKey(ticketId)) {
+            return tickets.get(ticketId);
+        } else {
+            throw new TicketNotFoundException("Ticket with such id not found");
+        }
+    }
 
+    public static void main(String[] args) {
         TicketService ticketService = new TicketService();
-        ticketService.save(emptyTicket);
-        ticketService.save(fullTicket);
-        ticketService.save(limitedTicket);
+        TicketBuilder.createTickets(ticketService);
 
-        System.out.println(ticketService.findAll());
 
+        System.out.println(ticketService.findById("2B"));
```

Comment:
For now it is ok, but for future, using System.out.println for logging in real apps is generally not recommended.